From 43369106b87a4f46876fdd43966368acc786b153 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 28 Aug 2014 03:04:11 -0400
Subject: [acpi] acpi_bind_one()/acpi_unbind_one() whitespace cleanups

Message-id: <20140828030411.2117.64568.stgit@gir.stowe>
Patchwork-id: 89294
O-Subject: [RHEL7.1 PATCH 06/30] ACPI: acpi_bind_one()/acpi_unbind_one() whitespace cleanups
Bugzilla: 1128632
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1128632
Upstream Status: f501b6ec290f59b9c444bc061acf0e422347fb55

commit f501b6ec290f59b9c444bc061acf0e422347fb55
Author: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Date:   Wed Aug 7 01:19:52 2013 +0200

    ACPI: acpi_bind_one()/acpi_unbind_one() whitespace cleanups

    Clean up some inconsistent use of whitespace in acpi_bind_one() and
    acpi_unbind_one().

    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
    Acked-by: Toshi Kani <toshi.kani@hp.com>
    Acked-by: Yasuaki Ishimatsu <isimatu.yasuaki@jp.fujitsu.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/glue.c b/drivers/acpi/glue.c
index 914a346..69641c0 100644
--- a/drivers/acpi/glue.c
+++ b/drivers/acpi/glue.c
@@ -247,9 +247,9 @@ int acpi_bind_one(struct device *dev, acpi_handle handle)
 
  acpi_physnode_link_name(physical_node_name, node_id);
  retval = sysfs_create_link(&acpi_dev->dev.kobj, &dev->kobj,
-   physical_node_name);
+       physical_node_name);
  retval = sysfs_create_link(&dev->kobj, &acpi_dev->dev.kobj,
-  "firmware_node");
+       "firmware_node");
 
  mutex_unlock(&acpi_dev->physical_node_lock);
 
@@ -293,12 +293,11 @@ int acpi_unbind_one(struct device *dev)
   char physical_node_name[PHYSICAL_NODE_NAME_SIZE];
 
   entry = list_entry(node, struct acpi_device_physical_node,
-   node);
+       node);
   if (entry->dev != dev)
    continue;
 
   list_del(node);
-
   acpi_dev->physical_node_count--;
 
   acpi_physnode_link_name(physical_node_name, entry->node_id);
-- 
1.7.1